compiler/transforms/stream_snaxify.py

```diff
@@ -130,24 +128,34 @@ def generate_one_list(n: int, i: int):
                     "Access patterns with symbols are not supported yet."
                 )
 
-            temp_dim = streamer_config.data.temporal_dim()
+            streamer_config.data.temporal_dim()
             spat_dim = streamer_config.data.spatial_dim()
 
+            # extremely dirty fix:
+            if spat_dim == 2:
+                spat_dim = 3
+
```

Comment:
This is the largest problem here, the access pattern of the matmul has 3 spatial for loops, but the gemm only has 2 spatial streamers, because on for loop is always irrelevant. I have no fix for this yet, just an idea how to fix it, with "virtual streamers". This only works because the gemm is the only one with a spat_dim of 2